### PR TITLE
[T2A4][T16-A3]Simaria Yesha Bharat

### PR DESCRIPTION
**Pull request by yeshasimaria**

### CODE REVIEW

**Comment by yeshasimaria** [comment]
Ready for review.

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -485,7 +485,7 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
     private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
-            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
+            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person).toLowerCase()));
```
Comment (sooyj) [inline src/seedu/addressbook/AddressBook.java:488]: Good code logic change.

File: test/input.txt [hunk 2]
```diff
@@ -51,7 +51,7 @@
   find bet
   # does not match if none have keyword
   find 23912039120
-  # matching should be case-sensitive
+  # matching should be case insensitive
```
Comment (sooyj) [inline test/input.txt:54]: Good work on updating the test input + expected output!

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -444,13 +444,13 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
 
     /**
      * Finds and lists all persons in address book whose name contains any of the argument keywords.
-     * Keyword matching is case sensitive.
+     * Keyword matching is case insensitive.
```
Comment (sooyj) [inline src/seedu/addressbook/AddressBook.java:447]: Correct! Comments should be updated too.

However, you may have missed the help message for this command :-). Look for COMMAND_FIND_DESC

**Review by sooyj — COMMENTED** [review]
Updated the "find" command to be case insensitive.

Overall good work, only missed the help message for the command. :-)

**Comment by yeshasimaria** [comment]
@sooyj Thank you! I have fixed the help message! :)

**Pull request closed without merging**